modelling/environmental_mod/index.md

```diff
@@ -20,59 +20,113 @@ parent: Modelling
+- CCSM4
+- CESM1_WACCM
+- GFDL_ESM2M
+- GISS_E2_R
+- HadGEM2
+- MIROC5
 
 # References
 
-Baird, M.E., Wild-Allen, K.A., Parslow, J., Mongin, M., Robson, B., Skerratt, J., Rizwi, F., Soja-Woźniak, M., Jones, E., Herzfeld, M. and Margvelashvili, N., 2020. CSIRO Environmental Modelling Suite (EMS): scientific description of the optical and biogeochemical models (vB3p0). Geoscientific Model Development, 13(9), pp.4503-4553. [Read Here](https://www.eprints.qut.edu.au/174674/8/Baird_et_al_2020_Geosci_Model_Dev.pdf)
+Baird, M.E., Wild-Allen, K.A., Parslow, J., Mongin, M., Robson, B., Skerratt, J., Rizwi, F., Soja-Woźniak, M., Jones, E., Herzfeld, M., & Margvelashvili, N. (2020). CSIRO Environmental Modelling Suite (EMS): scientific description of the optical and biogeochemical models (vB3p0). Geoscientific Model Development, 13(9), pp.4503-4553. https://doi.org/10.5194/gmd-13-4503-2020
+
+Callaghan, D. P., Leon, J. X., & Saunders, M. I. (2015). Wave modelling as a proxy for seagrass ecological modelling: Comparing fetch and process-based predictions for a bay and reef lagoon. Estuarine, Coastal and Shelf Science, 153, 108–120. https://doi.org/10.1016/j.ecss.2014.12.016
+
+Delandmeter, P., and Van Sebille, E. (2019). The Parcels v2. 0 Lagrangian framework: new field interpolation schemes. Geoscientific Model Development, 12(8), pp.3571-3584. https://doi.org/10.5194/gmd-12-3571-2019 
+
+McWhorter, J.K., Halloran, P.R., Roff, G., Skirving, W.J., Mumby, P. (2022) Climate refugia on the Great Barrier Reef fail when global warming exceeds 3°C. Glob Change Biology. 28, pp.5768–5780.
+https://doi.org/10.1111/gcb.16323 
+
+Roelfsema, C., Kovacs, E., Ortiz, J.C., Wolff, N.H., Callaghan, D., Wettle, M., Ronan, M., Hamylton, S.M., Mumby, P.J., Phinn, S. (2018). Coral reef habitat mapping: a combination of object-based image analysis and ecological modelling. Remote Sensing of Environment. 208, pp.27–41. http://dx.doi.org/10.1016/j.rse.2018.02.005
+
+Roelfsema, C.M., Kovacs, E.M., Ortiz, J.C., Callaghan, D.P., Hock, K., Mathieu, M., Johansen, K., Mumby, P., Wettle, M., Ronan, M., Lundgren, P., Kennedy, E.V., Phinn, S.R. (2020). Habitat maps to enhance monitoring and management of the Great Barrier Reef. Coral Reefs. 39, pp.1039–1054. https://doi.org/10.1007/s00338-020-01929-3
 
-Delandmeter, P. and Van Sebille, E., 2019. The Parcels v2. 0 Lagrangian framework: new field interpolation schemes. Geoscientific Model Development, 12(8), pp.3571-3584. [Read Here](https://gmd.copernicus.org/articles/12/3571/2019/)
+Steven, A.D., Baird, M.E., Brinkman, R., Car, N.J., Cox, S.J., Herzfeld, M., Hodge, J., Jones, E., King, E., Margvelashvili, N. and Robillot, C. (2019). eReefs: An operational information system for managing the Great Barrier Reef. Journal of Operational Oceanography, 12(sup2), pp.S12-S28. https://doi.org/10.1080/1755876X.2019.1650589 
```

Comment:
i think you left some authors out. suggest adding "et al." at the end of the authors, or list all authors.

Reply:
Confirmed - there are only two authors for reference.

Reply:
No I mean the eReefs paper. There are 18 co-authors on the paper (including Sharon, Barbara and myself omitted). Either list all or add et al. to the end. thanks!